99592 MainProcess 2023-04-04 07:15:39,985
Traceback (most recent call last):
  File "/Users/jondy/workspace/pyarmor/pyarmor-8.1.dev3/pyarmor/cli/__main__.py", line 627, in main
    main_entry(sys.argv[1:])
  File "/Users/jondy/workspace/pyarmor/pyarmor-8.1.dev3/pyarmor/cli/__main__.py", line 615, in main_entry
    args.func(ctx, args)
  File "/Users/jondy/workspace/pyarmor/pyarmor-8.1.dev3/pyarmor/cli/__main__.py", line 179, in cmd_gen
    builder.process(options, pack=args.pack)
  File "/Users/jondy/workspace/pyarmor/pyarmor-8.1.dev3/pyarmor/cli/generate.py", line 155, in process
    self._obfuscate_scripts()
  File "/Users/jondy/workspace/pyarmor/pyarmor-8.1.dev3/pyarmor/cli/generate.py", line 120, in _obfuscate_scripts
    code = Pytransform3.generate_obfuscated_script(self.ctx, r)
  File "/Users/jondy/workspace/pyarmor/pyarmor-8.1.dev3/pyarmor/cli/core/__init__.py", line 48, in generate_obfuscated_script
    return m.generate_obfuscated_script(ctx, res)
  File "<maker>", line 719, in generate_obfuscated_script
  File "<maker>", line 493, in process
  File "<rftmaker>", line 1
    8ï¿½ï¿½ï¿½lï¿½Ê¸ï¿½ï¿½tï¿½nï¿½ï¿½@Fï¿½Jï¿½ï¿½ï¿½ï¿½ï¿½`ï¿½$"<ï¿½ï¿½#?z]>Þ¤ï¿½cï¿½ï¿½ï¿½3ï¿½ï¿½xJï¿½ï¿½ï¿½ï¿½ï¿½Q8C0	7ï¿½ï¿½ï¿½rï¿½ ï¿½y
     ^
SyntaxError: invalid syntax
